modules/qna/src/translations/es.json

```diff
@@ -0,0 +1,102 @@
+{
+  "addNew": "Añadir nuevo",
+  "answer": "Respuesta",
+  "confirmDelete": "¿Quieres eliminar la pregunta?",
+  "context": {
+    "canTypeToCreate": "Puede escribir en la barra de selección para agregar nuevos contextos.",
+    "createQuery": "Crear contexto",
+    "filterByContexts": "Filtrar por contextos",
+    "selectContext": "Seleccionar contexto...",
+    "title": "Contextos"
+  },
+  "contexts": "Contextos",
+  "create": "Crear una nueva Q y A",
```

Comment:
please use the same terminology. Pick one and use it all across the app.
So far you have : `Q&A - QNA - QnA -  Q y A`

Reply:
got it, i will stick to Q&A